### DISCUSSION TITLE
wiki/help-pages/

### DISCUSSION THREAD

**Discussion by giscus** [PrismLauncher/prismlauncher.org#542] [Wiki]
# wiki/help-pages/

An Open Source Minecraft launcher with the ability to manage multiple instances, accounts and mods. Focused on user freedom and free redistributability.

https://prismlauncher.org/wiki/help-pages/

**Comment by argemozlr** [comment]
por favor alguien que me ayude, no se como solucionar el error Exception caught from launcher llevo horas intentado solucionarlo

**Comment by bankamer** [comment]
Не удается продолжить выполнение кода, поскольку система не обнаружила MSVCP140_1.dII. Для устранения этой проблемы попробуйте переустановить программу Не удается продолжить выполнение кода, поскольку система не обнаружила VCRUNTIME140.dII. Для устранения этой проблемы попробуйте переустановить программу.

**Reply by lordofpipes** [reply]
> Не удается продолжить выполнение кода, поскольку система не обнаружила MSVCP140_1.dII. Для устранения этой проблемы попробуйте переустановить программу Не удается продолжить выполнение кода, поскольку система не обнаружила VCRUNTIME140.dII. Для устранения этой проблемы попробуйте переустановить программу.

If you're experiencing the error

"The code execution cannot proceed because MSVCP140_1.dll was not found. Reinstalling the program may fix the problem."

or 

"The code execution cannot proceed because VCRUNTIME140_1.dll was not found. Reinstalling the program may fix the problem."

You can fix the issue by installing [Microsoft Visual C++ Redistributable 2022](https://learn.microsoft.com/en-us/cpp/windows/latest-supported-vc-redist?view=msvc-170#latest-microsoft-visual-c-redistributable-version)

**Reply by bankamer** [reply]
Я установил версии Microsoft Visual C++  с 2015-2022 год. Не заработало.

**Comment by AgentShadowStalker** [comment]
Everytime i try to create an new world it always do:

*failed to connect to server internal exception io.netty.handle.codec.EcoderException:Failed to encode packet 'serverbound/minecraft:hello'*

I have no idea why this keeps happening, do anyone knows an solution for this??

**Comment by BooTheRotten** [comment]
When I try to add my microsoft account, the error message 'Failed to get authorization for Xbox services: Operation cancelled' shows up when the progress bar shows 'logging in as an xbox user'. I do not know what is causing this and some help about this issue will be appreciated. I'm using the latest version of Prism on Mac with the univeral edition downloaded and not the legacy one.

**Comment by Astron34** [comment]
I have a problem i  cant sign in with my microsoft account its shows (Microsoft user authentication failed.)
